lte/gateway/c/oai/tasks/amf/amf_as.cpp

```diff
@@ -1164,11 +1164,10 @@ static int amf_as_security_req(
         if (ue_context) {
           amf_security_context_t* amf_security_context =
               &ue_context->amf_context._security;
-          amf_security_context->selected_algorithms.integrity = 0;
-          //   M5G_NAS_SECURITY_ALGORITHMS_128_5G_IA2;  // TODO get this
-          //   computed
-          amf_security_context->selected_algorithms.encryption = 0;
-          //  M5G_NAS_SECURITY_ALGORITHMS_5G_EA0;  // TODO get this computed
+          amf_security_context->selected_algorithms.integrity =
+              M5G_NAS_SECURITY_ALGORITHMS_128_5G_IA2;  // TODO get this computed
```

Comment:
non-increasing order of priority of the algorithms is 
EA2  EA1  EA0
IA2  IA1  IA0

of the UE security capabilities present in the UE Context 
pick the highest priority algorithm that AMF supports. can we fix this quickly.

Reply:
as part of this PR, the Default algorithm changed from IA0 to IA2.
I will raise separate user story and resolve order of priority of the algorithms